snuba/web/db_query.py

```diff
@@ -57,8 +57,9 @@ def update_query_metadata_and_stats(
 
 
 def raw_query(
-    request: Request,
+    physical_query: Query,
     query: ClickhouseQuery,
```

Comment:
Yeah, there are a few things we discussed yesterday that I did not yet have time to retrofit in this change.

About specifically passing two query objects. That pissed me off quite a bit. I tried to remove that duplication but:
1) if I pass only Query here, it means I need to bring over the logic that formats it. That is (for tracing reasons) in a context. Thus it means this whole method which is already unreadable, gets indented more.
2) if I pass only ClickhosueQuery it just does not work because this method depends on the physical query object.

I gave up only because this is not different than what we are already doing. The physical query is just embedded in the Rrequest object so we are already passing both.

Reply:
That's fine, I just couldn't remember if we actually discussed that and what the conclusion was if we did. As long as `ClickhouseQuery` eventually becomes a subtype of `PhysicalQuery` (or interchangeable with it) and we fix it at that point, that's fine with me. I just wanted to be sure that was actually the plan.

Reply:
now we have Clickhouse Query (query.clickhouse.Query)